migrations/addresses.json

```diff
@@ -331,6 +331,7 @@
     "syscoin": {
         "syscoin.testnet": {
             "SysUsdtPriceFeed": "0x4092CC222B1bD1176D8a8cBE4070ccea66E9d1D6"
+        "syscoin.mainnet": {
```

Comment:
Are we missing `0x3806311c7138ddF2bAF2C2093ff3633E5A73AbD4` here?

Reply:
Totally